app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/ArticleFragment.java

```diff
@@ -0,0 +1,162 @@
+package edu.grinnell.grinnell_publications_android.Fragments;
+
+
+import android.support.design.widget.CollapsingToolbarLayout;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v4.app.Fragment;
+import android.util.DisplayMetrics;
+
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.squareup.picasso.Picasso;
+import edu.grinnell.grinnell_publications_android.R;
+
+
+
+import java.util.Date;
+
+/**
+ * TODO: Fix getDrawable deprication
+ *
+ */
+
+public class ArticleFragment extends Fragment {
+
+    CollapsingToolbarLayout collapsingToolbar;
```

Comment:
You should make sure access modifiers are used here. Also, please use proper variable names. i.e this should read `private CollapisingToolbarLayout mCollapsingToolbar;`